Objects/frameobject.c

```diff
@@ -316,10 +324,17 @@ frame_setlineno(PyFrameObject *f, PyObject* p_new_lineno, void *Py_UNUSED(ignore
         delta--;
     }
 
+    if (view.obj != NULL)
```

Comment:
nit: The latest C code style recommends using `{}` always, even for single line clauses.